parl/remote/client.py

```diff
@@ -96,6 +99,15 @@ def read_local_files(self, distributed_files=[]):
         pyfiles['other_files'] = {}
 
         code_files = filter(lambda x: x.endswith('.py'), os.listdir('./'))
+        
+        parsed_distributed_files = []
+        for distributed_file in distributed_files:
+            parsed_list = glob.glob(distributed_file)
+            # exclude the directiories
+            for pathname in parsed_list:
+                if not os.path.isdir(pathname):
+                    parsed_distributed_files.append(pathname)
+        parsed_distributed_files = list(set(parsed_distributed_files))
```

Comment:
Why don't you declare `parsed_distributed_files` as a set?

Reply:
modified